src/terminal/adapter/ut_adapter/adapterTest.cpp

```diff
@@ -1378,6 +1400,8 @@ class TestGetSet final : public ConGetSet
     short _expectedColorTableIndex = -1;
     COLORREF _expectedColorValue = INVALID_COLOR;
 
+    bool _fPrivateSetDefaultColorResult = false;

```

Comment:
I'm not a fan of the overloaded usage of this. I think there's a boolean defined per callback mock with no sharing.

Reply:
I'll fix the duplicated use of _expectedColorValue too.
Are these things actually used anywhere?

Reply:
They might not be. When I stood up the original parser years ago and wrote the mock, every mock was checked somehow. Over time, it probably decayed and the only thing left is that the mock has to be implemented to satisfy compilation of the interface. I'll file a follow-on to try to improve the test coverage overall here.

Reply:
Now #987